discussion.md

```diff
@@ -0,0 +1,54 @@
+#Discussion 
+
+##1. Summary of PCA paper
+The question that PCA asks is "Is there another basis which is a linear combination of the original basis that best re-expresses our dataset?" (It is quoted from the paper but I think it best expresses what the analysis does) — @Sarah Majors 
+
+PCA stands for principal component analysis is a standard tool in modern data analysis because it is a simple, non-parametric method for extracting relevant information from confusing data sets. — @Marce
+
+##2. Can anyone explain why eigen vectors play an important role in calculating PCA???
```

Comment:
* Change eigen vectors->eigenvectors  (is one word)
* Only use one question mark at the end.